sunpy/map/tests/test_mapbase.py

```diff
@@ -1732,23 +1732,15 @@ def test_map_arithmetic_neg(aia171_test_map):
     check_arithmetic_value_and_units(new_map, -aia171_test_map.quantity)
 
 
-@pytest.mark.parametrize(("value", "warn_context"), [
-    ('map', pytest.warns(RuntimeWarning)),
-    ('foobar', contextlib.nullcontext()),
-    (None, contextlib.nullcontext()),
-    (['foo', 'bar'], contextlib.nullcontext()),
-])
-def test_map_arithmetic_operations_raise_exceptions(aia171_test_map, value, warn_context):
+@pytest.mark.parametrize("value", ['map', 'foobar', None, ['foo', 'bar']])
+def test_map_arithmetic_operations_raise_exceptions(aia171_test_map, value):
     value = aia171_test_map if value == 'map' else value
     with pytest.raises(TypeError):
         _ = aia171_test_map + value
```

Comment:
Do we need these returns?

Reply:
Well, no, hence the assignment to `_`, but I guess you're asking whether we can just delete the `_ =`?  Does a linter complain if we remove those?

Reply:
Yeah, can we just delete those returns? 

If the linter complains, it should be thrown out.